verification/curator-service/ui/src/components/landing-page/SignInForm.tsx

```diff
@@ -71,7 +71,9 @@ interface SignInFormProps {
     setRegistrationScreenOn: (active: boolean) => void;
 }
 
-const RECAPTCHA_SITE_KEY = process.env.RECAPTCHA_SITE_KEY as string;
+const RECAPTCHA_SITE_KEY = window.Cypress
+    ? '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'
```

Comment:
is this supposed to be secret? not sure

Reply:
Thank you for your feedback! That's a great question. It is a ReCaptchas site-key for testing purposes which is available for all developers: https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do Since every developer have access to it and it is the same for everyone I didn't set it as a secret.